docs/api-reference/next/image.md

```diff
@@ -48,15 +48,27 @@ When using an external URL, you must add it to
 
 ### width
 
-The width of the image, in pixels. Must be an integer without a unit.
+The width of the image. Must be an integer without a unit.
 
-Required, except for statically imported images, or those with [`layout="fill"`](#layout).
+The meaning of the `width` property can change depending on the [`layout`](#layout) and [`sizes`](#sizes) properties since some layouts will stretch to fill the parent container and others will not.
+
+The `width` property represents the _rendered_ width in pixels when `layout="intrinsic"`, `layout="fixed"`, or `layout="raw"` without `sizes` so it will affect how large the image appears.
+
+The `width` property represents the _original_ width in pixels when `layout="responsive"`, `layout="fill"`, or `layout="raw"` with `sizes` so it will only affect the aspect ratio.
+
```

Comment:
```suggestion
The `width` property also represents the _original_ width in pixels when `layout="responsive"`, `layout="fill"`, or `layout="raw"` are used _with_ `sizes`, so it will only affect the aspect ratio.
```

Reply:
@molebox I dont think `also` is the correct word here because its either one or the other. "also" implies both